pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -261,62 +269,99 @@ func (p *provider) Create(machine *v1alpha1.Machine, _ *cloudprovidertypes.Provi
 		return nil, err
 	}
 
-	quantity, err := resource.ParseQuantity("2Gi")
+	quantity, err := resource.ParseQuantity("10Gi")
```

Comment:
Instead of hardcoding this, can we add this to the config, similarily how we did id for the `sourceURL`?
Also please extend the `Validate` if the value of the field can be parsed by `resource.ParseQuantity`

Reply:
Yeah that's a good idea actually, will do ;)